3rdparty/optee/libutee/CMakeLists.txt

```diff
@@ -6,7 +6,8 @@ add_library(oeuteeasm
 set_property(TARGET oeuteeasm PROPERTY C_STANDARD 99)
 target_compile_options(oeuteeasm PRIVATE
   ${OE_TZ_TA_S_FLAGS}
-  -include ${OE_TZ_TA_DEV_KIT_CONF})
+  $<BUILD_INTERFACE:-include ${OE_TZ_TA_DEV_KIT_CONF}>
+  $<INSTALL_INTERFACE:-include openenclave/3rdparty/optee/conf.h>)
```

Comment:
These are private compile flags and will only be used when compiling `oeuteeasm`. I don't understand why an INSTALL_INTERFACE is necessary

Reply:
Mm, good point. Perhaps only the interface at the `oeutee` level is enough.

Reply:
Removed.